appengine-java8/urlfetch/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -2,4 +2,5 @@
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <runtime>java8</runtime>
   <threadsafe>true</threadsafe>
+  <url-stream-handler>urlfetch</url-stream-handler>
```

Comment:
This is only needed if you wish to use our service to do it, why do you need this?

Reply:
I read our documentation/error wrong...